src/functionalTest/resources/scenarios/RWA-340-Allocate-a-case-to-caseworker-negative-scenario-midevent.json

```diff
@@ -1,5 +1,6 @@
 {
   "description": "RWA-340 Allocate a case to caseworker when there is no assignments for the location midEvent",
+  "disabled": "true",
```

Comment:
same here

Reply:
i reset this test on master and it is not part of my PR anymore